src/pages/core/blog/core-js-sdk-v1.md

```diff
@@ -0,0 +1,312 @@
+---
+title: Core JS SDK V1
+metaTitle: Core JS SDK V1
+description: Whats new in the Core JS SDK V1?
+---
+
+## The V1 Milestone!
+
+Launching the **Core JS SDK V1** welcomes new improvements to both naming and functionality for devs and end users working with the JS Mpl Core package.
+
+## Major Changes
+
+### Plugin Constructor Functions
+
+Though while still present in the code base, the new wrappers allow for plugins to be defined without constructor functions.
+
+**Auto Generated Kinobi Functions**
+
+```ts
+await createV1(umi, {
+  asset: assetSigner,
+  name: 'My Asset',
+  uri: 'https://example.com/my-asset.json',
+  plugins: [
+    {
+      plugin: createPluginV2({
+        type: 'Attributes',
+        attributeList: [{ key: 'key', value: 'value' }],
+      }),
+      authority: pluginAuthority('UpdateAuthority'),
+    },
+  ],
+}).sendAndConfirm(umi)
+```
+
+**JS SDK V1**
+
+```ts
+await create(umi, {
+  asset: assetSigner,
+  name: 'My Asset',
+  uri: 'https://example.com/my-asset.json',
+  plugins: [
+    { type: 'Attributes', attributeList: [{ key: 'key', value: 'value' }] },
+  ],
+}).sendAndConfirm(umi)
+```
+
+### Plugin Data
+
+Plugin data is elevated to the top level instead of nested under the data field in a plugin object.
+
+**Auto Generated Kinobi Functions**
+
+```ts
+await addPluginV1(umi, {
+  asset: asset.publicKey,
+  plugin: createPlugin({ type: 'FreezeDelegate', data: { frozen: true } }),
+  initAuthority: addressPluginAuthority(delegate),
+}).sendAndConfirm(umi)
+```
+
+**JS SDK V1**
+
+```ts
+await addPlugin(umi, {
+  asset: assetId,
+  plugin: {
+    type: 'Attributes',
+    attributeList: [{ key: 'key', value: 'value' }],
+  },
+}).sendAndConfirm(umi)
+```
+
+### Lifecycle Wrapper now Requires Asset Objects
+
+The create/update/transfer/burn now require the full asset/collection objects in order to derive extra accounts if any.
```

Comment:
Could you just add a phrase or sentence here saying when there would be extra accounts?

Reply:
"The create/update/transfer/burn now require the full asset/collection objects in order to derive extra accounts (such as external plugin adapter accounts) if any if any."